ethsigner/app/src/main/java/tech/pegasys/ethsigner/CommandLineConfig.java

```diff
@@ -217,8 +210,22 @@ public String toString() {
         .add("chainId", chainId)
         .add("dataDirectory", dataDirectory)
         .add("output", output)
-        .add("hashicorpSignerConfig", hashicorpSignerBasedConfig)
-        .add("filebasedSignerConfig", fileBasedSignerConfig)
+        .add("hashicorpTransactionSignerConfig", hashicorpTransactionSignerConfig)
```

Comment:
If subcommands get removed from here, then this will need to go.

Reply:
As I said, I think I like having the CommandLineConfig as the place that holds all the configuration that is needed.